modules/virt-updating-imported-vm-network-name.adoc

```diff
@@ -3,20 +3,20 @@
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 
 [id="virt-updating-imported-vmware-vm-network-name_{context}"]
-= Updating the imported VMware virtual machine's NIC name
+= Updating the NIC name
```

Comment:
The title and the file name should align more closely. Maybe "Updating the NIC name of an imported virtual machine"?